### ISSUE TITLE
Datepicker with TimePicker: screenreader doesn't inform about changed time

### ISSUE THREAD

**Issue by geromegrignon** [openng-org/optimus-ui#627]
### Describe the bug

When using time picker and changing the time the screen reader is not informed.

### Reason for not contributing a PR

- [ ] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primeng.org/datepicker#time

### Environment

primeng v19
also tested and failing on: primeng v18 & primeng v17

### Angular version

19.x.x

### PrimeNG version

v19

### Steps to reproduce the behavior

1. Start Screenreader (I tested with NVDA)
2. Open Timepicker
3. Change Time with buttons
4. Screenreader will not read changed time

### Expected behavior

When changing the time screenreader should inform user about the changed time.

> Migrated from `primefaces/primeng#17455` — originally by `@mrkshx` on 2025-01-21

**Comment by geromegrignon** [comment]
*[Originally posted by `@github-actions[bot]` on 2025-04-16T14:04:54.000Z]*

Due to PrimeTek's demanding roadmap for PrimeNG, this issue is available for anyone to work on. [**Make sure to reference this issue in your pull request.**](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) :sparkles: Thank you for your contribution! :sparkles:
